config/manager/manager.yaml

```diff
@@ -102,5 +102,14 @@ spec:
             requests:
               cpu: 1000m
               memory: 512Mi
+          volumeMounts:
```

Comment:
Need SSL_CERT_DIR env var to be set in this file.

```
      containers:
        - command:
            - /manager
          ...
          env:
          ...
            - name: SSL_CERT_DIR
              value: /tls
              ...
```

Reply:
Tested this in custom environment, worked after this change.

Reply:
tested this as well and verified with this things are now working well in a custom environment